src/System.Globalization/tests/RegionInfo/RegionInfoTests.netstandard1.7.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Reflection;
+using Xunit;
+
+namespace System.Globalization.Tests
+{
+    public class RegionInfoMiscTests
+    {
+        public static IEnumerable<object[]> RegionInfo_TestData()
+        {
+            yield return new object[] { 0x409, 244, "US Dollar", "\u0055\u0053\u0020\u0044\u006f\u006c\u006c\u0061\u0072", "USA", "USA" };
+            yield return new object[] { 0x411, 122, "Japanese Yen", "\u5186", "JPN", "JPN" };
+            yield return new object[] { 0x804, 45, "Chinese Yuan", "\u4eba\u6c11\u5e01", "CHN", "CHN" };
+            yield return new object[] { 0x401, 205, "Saudi Riyal", "\u0631\u064a\u0627\u0644\u00a0\u0633\u0639\u0648\u062f\u064a", "SAU", "SAU" };
+            yield return new object[] { 0x412, 134, "South Korean Won", "\uc6d0", "KOR", "KOR" };
+            yield return new object[] { 0x40d, 117, "Israeli New Shekel", "\u05e9\u05e7\u05dc\u0020\u05d7\u05d3\u05e9", "ISR", "ISR" };
+        }
+
+        [Theory]
+        [PlatformSpecific(Xunit.PlatformID.Windows)] // Non Windows is not supported yet
+        [MemberData(nameof(RegionInfo_TestData))]
+        public void MiscTest(int lcid, int geoId, string currencyEnglishName, string currencyNativeName, string threeLetterISORegionName, string threeLetterWindowsRegionName)
+        {
+            RegionInfo ri = new RegionInfo(lcid); // create it with lcid
+            Assert.Equal(ri.GeoId, geoId);
+            Assert.Equal(ri.CurrencyEnglishName, currencyEnglishName);
+            Assert.Equal(ri.CurrencyNativeName, currencyNativeName);
+            Assert.Equal(ri.ThreeLetterISORegionName, threeLetterISORegionName);
+            Assert.Equal(ri.ThreeLetterWindowsRegionName, threeLetterWindowsRegionName);
+        }
+
+        [Fact]
+        [PlatformSpecific(Xunit.PlatformID.Windows)] // Non Windows is not supported yet
+        public void NegativeTest()
+        {
+            Assert.Throws<System.ArgumentException>("name", () => new RegionInfo(""));
```

Comment:
Oh, I see that is already in RegionInfoTests.Methods.cs

---
In reply to: [78286978](https://github.com/dotnet/corefx/pull/11594#discussion_r78286978) [](ancestors = 78286978)